templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -213,22 +299,62 @@ Resources:
           "{{ hour }}": "on"
 {% endfor %}
 {% endif %}
-      Hostname: !Sub "${ApplicationId}-web2"
+
+#OpswxEC2Web2PM ~ OpswxEC2Web6PM
+#OpswxEC2Web2AO ~ OpswxEC2Web6AO
+{% set subnetLetters = ["C","A","B"] %}
+{% for loopingTimes in range(2, 7) %}
+  OpswxEC2Web{{ loopingTimes }}{% if item.tags["PowerManaged"] == "Yes" %}PM{% else %}AO{% endif %}:
```

Comment:
This should be unchanged for non-power-managed instances so it doesn't shut down PRD.

Reply:
when you are working with always on instances, cloudformation will spin up all new resources and once they are in green state, then it will start to dismantle the old ones. 

This is very different on swapping between timed vs always on. Since there is no resource connect to timed (in cloudformations view) it just deletes the ops works instance without waiting of it (it does not check the clocks and wait if it is actually meant to be started etc)